src/zino/zino.py

```diff
@@ -35,7 +36,11 @@ def main():
         level=logging.INFO if not args.debug else logging.DEBUG,
         format="%(asctime)s - %(levelname)s - %(name)s (%(threadName)s) - %(message)s",
     )
-    state.state = state.ZinoState.load_state_from_file() or state.ZinoState()
+    state.config = read_configuration(args.config_file.name)
+    state.state = (
+        state.ZinoState.load_state_from_file(state.config.get("persistent_file", "zino-state.json"))
```

Comment:
Really, we shouldn't re-define the default value here, when it's already defined as `zino.state.STATE_FILENAME`